### DISCUSSION TITLE
Predicting thermal conductivites

### DISCUSSION THREAD

**Discussion by rfhari** [MPA2suite/k_SRME#10] [Q&A] [answered]
Hi, I’m not sure if this is the right place to post my question - I couldn’t find a more suitable option. 

Thank you for sharing this great work!

I have a quick question: phono3py typically calculates FC2 and FC3 using the forces (finite differences method) on atoms in displaced supercells. But these displacements are much smaller than those derived from MD trajectories at finite temperatures. Consequently, the corresponding force errors on displaced supercells tend to be much smaller than those on MD snapshots at finite temperature. I think this difference can potentially lead to an underestimation of Green-Kubo-based thermal conductivities from MD, while lattice thermal conductivities might still align more closely with experimental observations.

Given this context, could k_SRME​ sometimes be misleading for finite-temperature MD simulations? Sorry I don’t mean to point out errors - but I’m just a bit confused, as I’m facing a similar underestimation issue in my study on AlGaN while using ACE as a surrogate for predicting thermal conductivities in MD-GK method.

It will be great if you can share your thoughts on this. 

Thank you!

**Comment by MSimoncelli** [comment, marked as answer]
Dear rfhari,

In practice, what you mean by ‘finite-temperature’ strongly depends on the material. The higher the vibrational frequencies of a solid, the higher the temperature needed to enter the classical equipartition regime, where Green-Kubo methods do not require phenomenological corrections.

As you said, k_SRME compares finite displacements for both MLP and DFT around the equilibrium positions. From a physics viewpoint, this means that k_SRME probes the accuracy of fMLP in describing the properties of solids in the close-to-equilibrium regime.
As mentioned on the Matbench website, k_SRME provides complementary information to the F1 score, which quantifies the accuracy of fMLP in describing the equilibrium properties of solids (specifically the structure-energy relationship).

Describing the accuracy of fMLP in the far-from-equilibrium regime is a different challenge. Some tests related to this are already available on the MLIP-Arena: https://huggingface.co/spaces/atomind/mlip-arena.

Ultimately, the answer to your question is both system- and temperature-dependent. In Figure 5 of our manuscript (http://arxiv.org/abs/2408.00755), we show the example of LiBr, where predictions from the Wigner transport equation (on which k_SRME is based) at room temperature are compatible with those obtained from Green-Kubo.

**Reply by rfhari** [reply]
Thanks for the detailed explanation and additional context! I understand now.
